stock_picking_restrict_cancel_with_orig_move/tests/test_restrict_cancel.py

```diff
@@ -51,3 +51,28 @@ def test_restrict(self):
         self.input_to_qc_picking.action_cancel()
         self.assertEqual(qc_to_stock_move.state, 'cancel')
         self.assertEqual(self.input_to_qc_picking.move_lines.state, 'cancel')
+
+    def test_do_not_restrict(self):
+        self.env['stock.picking'].create({
```

Comment:
a comment on the pre-conditions on why this should work as tested would be nice :)